RELEASE-NOTES.txt

```diff
@@ -10,6 +10,8 @@
 * Block editor: Images clickable for fullscreen preview
 * Fixed time displayed on Post Conflict Detected and Unpublished Revision dialogs
 * Block editor: Fix issue when removing image/page break block crashes the app
+* Removed support for Giphy
```

Comment:
Should this update for Giphy be included here? Let me know 🙏

Reply:
Gah! That's from `develop`. Thanks for noticing that. 😞

Reply:
Fixed in db5472a758dc32ad8af3546fa035fbb9cb4c31a1